xmppserver/src/main/java/org/jivesoftware/openfire/muc/MUCRoom.java

```diff
@@ -65,6 +65,7 @@
  */
 @JiveID(JiveConstants.MUC_ROOM)
 public class MUCRoom implements GroupEventListener, UserEventListener, Externalizable, Result, Cacheable {
+    public static final boolean BULK_MSG_RETRACTION_ENABLED = JiveGlobals.getBooleanProperty("chatManagement.bulkMsgRetractionEnabled", true);
```

Comment:
Instead of `JiveGlobals` can we use `SystemProperty` please? That has various benefits (better type-safety, being self-descriptive, etc). An example of a boolean-based SystemProperty can be found in https://github.com/igniterealtime/Openfire/blob/a446aa55d936ebb7ed016ac889cb91917e3fd12a/xmppserver/src/main/java/org/jivesoftware/openfire/muc/MUCRoom.java#L71-L75

Reply:
Done.